package.json

```diff
@@ -136,10 +136,12 @@
     "ws": "^8.17.1"
   },
   "resolutions": {
-    "dompurify": "3.2.4",
-    "@types/dompurify": "3.0.5",
     "@sentry/core": "9.27.0",
-    "@sentry/types": "9.27.0"
+    "@sentry/types": "9.27.0",
+    "@sentry/nextjs": "9.27.0",
```

Comment:
hrm why wasnt nextjs already in here? i think this package.json is actually for the main docs site?